src/components/CheckBoxTree/index.tsx

```diff
@@ -99,6 +99,18 @@ class CheckBoxTree extends React.Component<CheckBoxTreeProps> {
         }
     };
 
+    getAgentTags = (agentName: string): string[] => {
+        const { treeData } = this.props;
+        const tags: string[] = [""];
+        const agent = treeData.find((agent) => agent.title === agentName);
+        if (agent) {
+            agent?.children.forEach((child) => {
```

Comment:
this `?` doesn't seem necessary if you're also checking for `if (agent)`

Reply:
you're right